enterprise/migrations/0045_report_type_json.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-05-09 16:08
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('enterprise', '0044_reporting_config_multiple_types'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='enterprisecustomerreportingconfiguration',
+            name='report_type',
+            field=models.CharField(choices=[('csv', 'csv'), ('json', 'json')], default='csv', help_text='The type this report should be sent as, e.g. CSV.', max_length=20, verbose_name='Report Type'),
```

Comment:
do we want to default to json?

Reply:
Current configurations are all sending out CSV files by default, so I assume most of the use cases are for CSV. But since we already set a default for this field in the previous migration, it wouldn't affect previous ones to change the default here. Up to @mattdrayer for what is more convenient.

Reply:
I don't think it matters, since the delivery format is going to be part of the conversation we have with each customer, anyway.  We can leave it as-is for now, if that's easier, or switch to JSON.

Reply:
:+1: lets leave it as is then.